doc/man/heketi-cli.8

```diff
@@ -29,37 +29,37 @@ Command line program for Heketi
 \fBheketi\-cli cluster create\fP
 Create a cluster
     \fBExample\fP
-    $ heketi-cli cluster create
+    $ heketi\-cli cluster create
```

Comment:
These are fixes that should imho go into a separate patch: the don't fit under the commit message of "add all new commands".